pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ReaderConfigurationData.java

```diff
@@ -67,6 +68,13 @@
 
     private transient List<ReaderInterceptor<T>> readerInterceptorList;
 
+    // max pending chunked message to avoid sending incomplete message into the queue and memory
+    private int maxPendingChunkedMessage = 10;
```

Comment:
Where do we pass these configurations to the internal consumer?

Reply:
it's configured in `ReaderImpl` and `MultiTopicsReaderImpl`